test/alternator/test_cql_rbac.py

```diff
@@ -369,3 +369,102 @@ def test_rbac_updateitem_read(dynamodb, cql, test_table_s):
                     UpdateExpression='SET v =  v + :val',
                     ExpressionAttributeValues={':val': 1})
     assert {'p': p, 'v': v2 + 1} == test_table_s.get_item(Key={'p': p}, ConsistentRead=True)['Item']
+
+# Test DeleteTable's permissions checks. The DeleteTable operation requires
+# a DROP permission on the specific table (or on something which contains it,
+# the keyspace all ALL KEYSPACES).
```

Comment:
> (or on something which contains it, the keyspace all ALL KEYSPACES).

I'm not sure I understand the part that comes after the `,`

Reply:
I think I wanted to write "or something which contains it, such as the keyspace or ALL KEYSPACES"
I think "all" was a typo for "or".

Reply:
Fixed.